internal/k8s/controller.go

```diff
@@ -1652,6 +1664,15 @@ func (lbc *LoadBalancerController) syncService(task task) {
 			if err != nil {
 				glog.V(3).Infof("error updating VirtualServer/VirtualServerRoute status in syncService: %v", err)
 			}
+
+			transportServers := lbc.configuration.GetResourcesWithFilter(resourceFilter{TransportServers: true})
```

Comment:
I think this part could be removed as well, considering IP will not be reported.
because in syncService we can only trigger status update related to the update in the IP

Also, I mentioned in this comment https://github.com/nginxinc/kubernetes-ingress/pull/1425/files#r587905227 about resourceFilter{TransportServers: true}, because GetResourcesWithFilter doesn't read that value (more over, it doesn't return TransportServers at all). If you remove this part, I suggest you remove that field in the filter as well

Reply:
done